### DISCUSSION TITLE
Multiple Prison Shacks

### DISCUSSION THREAD

**Discussion by RickJamesBish** [NeoArmageddon/co10_Escape#128] [Ideas]
May not be feasilble but it would be interesting to have multiple prison shacks. When you play with others, you can have one group in each shack. One group escapes and the other must get freed by that group, then they all escape the island together.

**Comment by CaptainPStar** [comment]
We got the same idea a while back. It might be good for large groups of 8-10 players but we rejected that idea because of two problems.

1. AI and server won't handle two seperate starts very well. For example the guards turn hostile, if a player escapes. How do you treat two prisons? Make all guards hostile? Players could be shoot before grabbing a gun. Seperate the logic of the hostility between the prisons: What happens if one group escapes and the other not. Hostility is global.
2. And a gameplay issue: One group is shot down completly. How can the other group come to revive? Maybe they are kilometers apart. Fun for the downed group will be ->0.

**Reply by RickJamesBish** [reply]
Ah ok. I didn't think about the AI freaking out. Plus the AI tend to go hostile before you even attempt escape.

**Comment by wismerhill59** [comment]
hello, I come back to this idea... two teams, each in different prison.  one objective : escape of both teams... communication with the radios and regroupment... really so difficult or impossible??? 
For the gameplay issue ... One group is shot down completly. -> one marker created and the other group try to come to revive (with vehicle, why not...)
:D
